packages/core/upload/server/services/api-upload-folder.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { isNil, get } = require('lodash/fp');
+const { getService } = require('../utils');
+
+const folderModel = 'plugin::upload.folder';
+const API_UPLOAD_FOLDER_BASE_NAME = 'Uploads';
+
+const getStore = () => strapi.store({ type: 'plugin', name: 'upload', key: 'api-folder' });
+
+const createApiUploadFolder = async () => {
+  let name = API_UPLOAD_FOLDER_BASE_NAME;
+  const folderService = getService('folder');
+
+  let exists = true;
+  let index = 1;
+  while (exists) {
+    exists = await folderService.exists({ name, parent: null }); // TODO: check if works
```

Comment:
Is the TODO resolved already?

Reply:
yes ahah I will remove it

Reply:
As mentioned in Slack, "API Uploads" rather than "Uploads" works for me :)